tests/requests/api/v1/favorites/index.spec.js

```diff
@@ -0,0 +1,45 @@
+var shell = require('shelljs');
+var request = require("supertest");
+var app = require('../../../../../app');
+
+const environment = process.env.NODE_ENV || 'test';
+const configuration = require('../../../../../knexfile')[environment];
+const database = require('knex')(configuration);
+
+
+describe('Test the favorites path', () => {
+  beforeEach(async () => {
+    await database.raw('truncate table favorites cascade');
+
+    let favoriteData = {
+      "title": "We Will Rock You",
+      "artistName": "Queen",
+      "genre": "Rock",
+      "rating": 88
+    }
+    await database('favorites').insert(favoriteData, 'id');
```

Comment:
Great creation of records for testing. We will probably need one more Favorite record to truly verify we're being returned all records.